debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -1158,4 +1198,24 @@ public static int validateRacNodes(Configuration config, Field field, Validation
         }
         return errors;
     }
+
+    private static int validateLogMiningBufferType(Configuration config, Field field, ValidationOutput problems) {
+        final LogMiningBufferType bufferType = LogMiningBufferType.parse(config.getString(LOG_MINING_BUFFER_TYPE));
```

Comment:
HotRod variant should enforce presence of hotrod client config params like `server`